stellargraph/layer/gcn_lstm.py

```diff
@@ -191,47 +193,43 @@ def call(self, features):
-        kernel_constraint (str or func, optional): The constraint to use for the weights of each layer.
-        bias_initializer (str or func, optional): The initialiser to use for the bias of each layer.
-        bias_regularizer (str or func, optional): The regulariser to use for the bias of each layer.
-        bias_constraint (str or func, optional): The constraint to use for the bias of each layer.
-    """
+        GraphConvolutionLSTM is a univariate timeseries forecasting method. The architecture  comprises of a stack of N1 Graph Convolutional layers followed by N2 LSTM layers, a Dropout layer, and  a Dense layer.
+        This main components of GNN architecture is inspired by: T-GCN: A Temporal Graph Convolutional Network for Traffic Prediction
+                                          (https://arxiv.org/abs/1811.05320)
+        The implementation of the above paper is based on one graph convolution layer stacked with a GRU layer.
+        The StellarGraph implementation is built as a stack of the following set of layers:
+           1. User specified no. of Graph Convolutional layers
+           2. User specified no. of LSTM layers
+           3. 1 Dense layer
+           4. 1 Dropout layer
+           The last two layers consistently showed better performance and regularization experimentally.
+       Args:
+           seq_len: No. of LSTM cells
+           adj: unweighted/weighted adjacency matrix of [no.of nodes by no. of nodes dimension
+           gc_layer_size (list of int): Output sizes of Graph Convolution  layers in the stack. 
+           lstm_layer_size (list of int): Output sizes of LSTM layers in the stack.
+           bias (bool): If True, a bias vector is learnt for each layer in the GCN model.
+           dropout (float): Dropout rate applied to input features of each GCN layer.
+           gc_activations (list of str or func): Activations applied to each layer's output;
+                defaults to ['relu', ..., 'relu'].
+           lstm_activations (list of str or func): Activations applied to each layer's output;
+                defaults to ['tanh', ..., 'tanh'].
+           kernel_initializer (str or func, optional): The initialiser to use for the weights of each layer.
+           kernel_regularizer (str or func, optional): The regulariser to use for the weights of each layer.
+           kernel_constraint (str or func, optional): The constraint to use for the weights of each layer.
+           bias_initializer (str or func, optional): The initialiser to use for the bias of each layer.
+           bias_regularizer (str or func, optional): The regulariser to use for the bias of each layer.
+           bias_constraint (str or func, optional): The constraint to use for the bias of each layer.
+        """
 
     def __init__(
         self,
         seq_len,
         adj,
-        gc_layers,
+        gc_layer_size,
```

Comment:
seeing as this list of layer sizes i think:
```suggestion
        gc_layer_sizes,
```

might make more sense. and the same below